Makefile

```diff
@@ -32,6 +32,7 @@ clean:
 	rm -rf python/az/aro/{aro.egg-info,build,dist} aro
 	find python -type f -name '*.pyc' -delete
 	find python -type d -name __pycache__ -delete
+	find -type d -name 'gomock_reflect_[1-9]*' -exec rm -rf {} \+ 2>/dev/null
```

Comment:
will these generated directories ever start with `0` as the first digit?

Reply:
@s-amann That is a great question, I created a simple function to test this and it turns out and indeed the generated directories can start with 0. I'll fix this in a commit 

```
package main

import (
	"fmt"
	"io/ioutil"
)

func main() {
	var dir = "<my_dir>"
	tmpDir, err := ioutil.TempDir(dir, "gomock_reflect_")
	if err != nil {
		fmt.Print(err)
	}
	fmt.Print(tmpDir)
}
/home/kmagdani/go/src/github.com/Azure/test/gomock_reflect_021545020
```